src/modules/cmdpal/Microsoft.CmdPal.UI/Microsoft.CmdPal.UI.csproj

```diff
@@ -45,6 +45,7 @@
     <PackageReference Include="CommunityToolkit.WinUI.Controls.Primitives" />
     <PackageReference Include="CommunityToolkit.WinUI.Converters" />
     <PackageReference Include="CommunityToolkit.WinUI.Extensions" />
+    <PackageReference Include="CommunityToolkit.WinUI.UI.Controls.Markdown" />
```

Comment:
If we can reference the lab feed, then we should try using the new one, see: https://github.com/zadjii-msft/PowerToys/pull/202/files#r1876720104